public/locales/en.json

```diff
@@ -15,7 +15,7 @@
   "input": "Input",
   "output": "Output",
   "estimated": "estimated",
-  "balance": "Balance: {{ balanceInput }}",
+  "balance": "Balance: <0>{{ balanceInput }}</0>",
```

Comment:
what does `<0>` mean?

Reply:
It's a placeholder to inject any HTML/ReactNode like: `<strong>{{ balanceInput }}</strong>`